onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -1593,16 +1593,51 @@ def aten_upsample_bicubic2d_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::upsample_bilinear2d")
 def aten_upsample_bilinear2d(
-    self: TensorType,
+    self: TReal,
     output_size: INT64,
-    align_corners: bool,
```

Comment:
Out of curiosity, why can we unfollow aten signature?

Reply:
If that's the case, scales_h and scales_w should be also brought back.

Reply:
They are in another overload function. Because output_size can scales_ cannot both show.